slacker.py

```diff
@@ -3,11 +3,11 @@
 # Make sure v3+ is used.
 import sys
 vinfo = sys.version_info
-if vinfo.major < 3:
-  print("You are using Python v{}.{}. v3+ is required!".format(vinfo.major, vinfo.minor))
+if vinfo < (3, 0):
+  print("Python v3+ is required! You are using v{}.{}.".format(vinfo.major, vinfo.minor))
   sys.exit(-1)
 
-from slacker.main import start_slacker
+from slacker.main import start_slacker  # noqa: E402
```

Comment:
This is done to ignore the error:
> E402 module level import not at top of file

We _have_ to have the version check first.

Reply:
Do we know which features we are using that explicitly require Python 3?

Reply:
We should actually try to map that information so we know what our exact minimum version requirement is.